database/factories/PageFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Page>
+ */
+class PageFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'title' => $this->faker->sentence,
+            'content' => $this->faker->paragraph,
+            'slug' => $this->faker->unique()->slug,
+            'published_at' => now(),
+            'author_id' => 'eaf9efc2-adbb-4b27-b5a9-f6c60197ab56', // Hardcoded to test user
```

Comment:
```suggestion
            'author_id' => User::factory(),
```

`author_id` can still be overridden if needed